fe/src/stores/useMailStore.ts

```diff
@@ -21,27 +21,36 @@ export const useMailStore = defineStore('user/mail', {
         mails: new Map(),
     }),
     actions: {
-        async loadStudentsMails() {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        async loadStudentsMails(pagination: any, setNumberOfRows: any) {
             this.isLoading = true
             const studentStore = useStudentStore()
 
             const filters = []
             if (this.searchMails) filters.push(`search=${this.searchMails}`)
+            const order = pagination.descending ? '-' : '+'
+            if (pagination.sortBy === 'name') {
+                filters.push(`ordering=${order}first_name,${order}last_name`)
+            } else if (pagination.sortBy !== null) {
+                filters.push(`ordering=${order}${pagination.sortBy}`)
+            }
 
             let url = ''
-            if (filters) url = `?${filters.join('&')}`
+            if (filters) url = `&${filters.join('&')}`
```

Comment:
Axios supports query params, it is better to use this instead of putting it in the url.

Reply:
Do you mean like this body parameter here?
![image](https://user-images.githubusercontent.com/57808740/167273213-34d591da-1d4b-4985-a479-838da21826ed.png)

Reply:
I'm not 100% certain, but it seems so yeah. To be sure you could do the request like this:

```
instance.get(url, params: {
	key: value,
})
```